css-layout-classes/styles.css

```diff
@@ -0,0 +1,41 @@
+  * {
+    font-family: Arial, Helvetica, sans-serif;
+    margin: 5px 10px;
+  }
+
+  p {
+    margin: 10px 10px;
+  }
+
+  img {
+    width: 100%;
+  }
+
+  .row {
+    display: flex;
+    justify-content: center;
+  }
+
+  .column-full {
+    display: flex;
+    justify-content: flex-start;
+    min-width: 100%;
+  }
+
+  .column-one-third {
+    flex-basis: calc(100% / 3);
+  }
+
+  .column-one-fourth {
+    min-width: 25%;
```

Comment:
I would replace this with `width` min-width is a limit, not a explicit declaration.